Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -178,6 +179,7 @@ data DriverEntityRes = DriverEntityRes
     enabled :: Bool,
     blocked :: Bool,
     verified :: Bool,
+    aadhaarVerified :: Bool,
     registeredAt :: UTCTime,
```

Comment:
we were planning to add `aadhaarVerificationRequired` field in profile API response as well, can we add it ?

Reply:
yes
I have added in merchant